Packs/FeedMISPThreatActors/Integrations/FeedMISPThreatActors/FeedMISPThreatActors_description.md

```diff
@@ -0,0 +1,13 @@
+# MISP Threat Actors Feed Integration
+
+## Overview
+
+The MISP Threat Actors Feed integration allows you to retrieve and ingest threat actor information from the MISP threat actor galaxy into your TIM.
```

Comment:
The MISP Threat Actors Feed integration allows you to retrieve and ingest threat actor information from the MISP threat actor galaxy into Cortex XSOAR Threat Intel.